### PR TITLE
Investigate orders load speed

### PR DESCRIPTION
**Pull request by W3stside**
Addresses several issues:
1. data load speed on xDAI (ordersWidget)
2. isListReady, necessary for not showing stale data, mixed with useEffect logic was blocking updates
3. types

### CODE REVIEW

File: src/services/factories/tokenList.ts [hunk 1]
```diff
@@ -312,16 +308,20 @@ export function getTokensFactory(factoryParams: {
...
     }
   }
 
   return function (networkId: number): TokenDetails[] {
     if (!areTokensUpdated.has(networkId)) {
       logDebug(`[tokenListFactory][${networkId}] Will update tokens for network`)
+
+      // Set token list readiness to false
+      // and prevent stale data being presented in app
+      tokenListApi.setListReady(false)
```
Comment (Velenir) [inline src/services/factories/tokenList.ts:320]: Should this not be awaited?

**Review by Velenir — APPROVED** [review]

File: src/services/factories/tokenList.ts (hunk 1 above)
Comment (W3stside) [reply src/services/factories/tokenList.ts:320]: why? its not async

Comment (Velenir) [reply src/services/factories/tokenList.ts:320]: Wrong line, updateTokens function

Comment (W3stside) [reply src/services/factories/tokenList.ts:320]: ah shit

**Pull request merged**